flow/Makefile

```diff
@@ -541,7 +537,7 @@ $(RESULTS_DIR)/1_synth.rtlil:
 	$(UNSET_AND_MAKE) do-yosys-canonicalize
 
 $(RESULTS_DIR)/1_1_yosys.v: $(RESULTS_DIR)/1_synth.rtlil
-	$(UNSET_AND_MAKE) do-yosys
+	$(UNSET_AND_MAKE) do-yosys-flatten-policy do-yosys
```

Comment:
It looks like SYNTH_HIERARCHICAL is no longer supported.  Is that correct and if so, why?

Reply:
Still supported, but the iffy code from Makefile is gone. It is all handled in .tcl now.